api_server/src/request/mmds.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use request::Body;
+use request::Error;
+use request::ParsedRequest;
+
+pub fn parse_get_mmds() -> Result<ParsedRequest, Error> {
```

Comment:
I would drop `instance_info.rs` and `mmds.rs` as separate modules and just treat them in `requests`.
1. They are very small (under 10 lines of production code),
2. Both MMDS and InstanceInfo are ApiServer-local resources.

@firecracker-microvm/compute-capsule what do you say?

Reply:
I like the separation approach better, but this is ultimately a highly subjective matter.

Reply:
Ok, keep them as they are.